test/e2e/Tests/start-emulators.ps1

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+param(
+    [Parameter(Mandatory=$false)]
+    [Switch]
+    $SkipStorageEmulator,
+    [Parameter(Mandatory=$false)]
+    [Switch]
+    $StartCosmosDBEmulator,
+    [Parameter(Mandatory=$false)]
+    $EmulatorStartDir,
+    [Parameter(Mandatory=$false)]
+    [Switch]
+    $NoWait
+)
+
+if (Test-Path($EmulatorStartDir)) {
+    Set-Location $EmulatorStartDir
+}
+
+$DebugPreference = 'Continue'
+
+Write-Host "Start CosmosDB Emulator: $StartCosmosDBEmulator"
+Write-Host "Skip Storage Emulator: $SkipStorageEmulator"
+
+$startedCosmos = $false
+$startedStorage = $false
+
+# Reassigning $IsWindows no longer allowed - might need to refactor this logic
+# if (!$IsWindows -and !$IsLinux -and !$IsMacOs)
+# {
+#     # For pre-PS6
+#     Write-Host "Could not resolve OS. Assuming Windows."
+#     $IsWindows = $true
+# }
+
+if (!$IsWindows)
```

Comment:
Consider adding `-and $StartCosmosDBEmulator` so that we don't write this message about skipping if the caller never asked for the cosmos DB emulator.

Reply:
Done